### PR TITLE
tutorial on writing user stories and cos

### PR DESCRIPTION
**Pull request by abhutta0**
starting with fa23 version

### CODE REVIEW

**Comment by mwand** [comment]
At end of first para:

"A user story is always of the following form: "

otherwise lgtm.

File: tutorials/week1-user-stories.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+layout: page
+title: User Stories and Conditions of Satisfaction
+permalink: /tutorials/week1-user-stories
+parent: Tutorials
+nav_order: 3
+---
+This tutorial provides examples for user stories, consitions of satisafation and how a minimum viable product can be defined.
```
Comment (PrajwalMathad) [inline tutorials/week1-user-stories.md:8]: Spelling correction: conditions of satisfaction

File: tutorials/week1-user-stories.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+### User Story #4: 
+As a player in covey.town, I want to learn different strategies for connect4, so I can win more games. (Extension)
+
+### User Story #5: 
+As a user of Covey.Town, I want to be able to interact with other users in different activities like private chat, playing games, etc. while providing ability to customize user avatars to make the town more fun.  (Essential but too general. Best to split it in 3 different stories one for private chat, one for games and one for customization.)
+
+## Conditions of Satisfaction:
+### For User Story #2:
+* There should be a Conncect4 area in the town, where players can play Connect4. (Essential)
```
Comment (PrajwalMathad) [inline tutorials/week1-user-stories.md:74]: Spelling correction: Connect4

**Comment by Muskaan-Nandu** [comment]
Apart from the 2 spell checks, everything else looks good.

**Review by Muskaan-Nandu — COMMENTED** [review]
Apart from the 2 spelling errors pointed out, everything else looks good to me.

**Comment by abhutta0** [comment]
both of you have a pending invitation for repo access. Accept and then you will have write access. Then please make the updates and confirm

**Comment by sindhukrovvidi** [comment]
Apart from the spelling mistakes mentioned, everything looks good to me.

**Pull request merged**